deploy/operator.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubeflow-operator
+  namespace: operators
```

Comment:
Is there a reason why the `namespace` is hardcoded in all manifests? Would it make sense to remove it so that user can deploy to any preferred namespace easily?

Reply:
Thanks @vpavlin, I updated all the manifests to exclude the namespace field now.